README.md

```diff
@@ -31,11 +31,18 @@ forge build
 3. Run tests:
 
 ```bash
-forge test
+forge test -vvv
```

Comment:
add `forge install ds-test` if we want to use the test.sol from lib/

Reply:
gotcha I'll add `git submodule update --init` which is a bit more generic in case we add more submodules later down the line